examples/relativistic/possibility_of_light_radiation_of_free_electron.py

```diff
@@ -62,25 +62,37 @@
 print("Energy conservation equation:")
 print(print_expression(energy_conservation_eqn))
 
-# We know that the Lorentz factor is always greater than 1, so we can use that information to figure
-# out if the above equation ever holds true.
+inequalities = energy_conservation_eqn & (lorentz_factor > 1)
 
-electron_rest_mass_range = reduce_inequalities(
-    energy_conservation_eqn & (lorentz_factor > 1),
-    [electron_rest_mass],
-)
+# The above inequalities never hold, so `reduce_inequalities` returns `False`
 
-# The above inequality never holds, therefore `reduce_inequalities` returns `False`
-assert electron_rest_mass_range == False
+assert reduce_inequalities(inequalities) == False
 
 # This indicates that free electrons are unable to radiate photons. Only accelerated electrons can do that,
 # for example, when the electron is in the vicinity of the atomic nucleus or is interacting with the electric
 # field.
 
-# What is more, we can use the same expressions that we derived here to show that free electrons cannot
-# absorb light either: `total_energy_after_radiation` gives the energy of the electron + photon system before
-# the absorption has happened in a reference frame where the total momentum of the system is zero, and
-# `total_energy_before_radiation` gives the energy of the electron after photon absorption.
+# Let us now consider the case of photon getting absorbed by the electron. We assume the reference frame
+# where the total momentum of the electron + photon system is zero.
+
+total_energy_before_absorption = moving_electron_energy + photon_energy
+
+# The total momentum is zero before and after absorption, therefore the electron is at rest after absorption.
```

Comment:
Not formally justified. We could prove it formally, but it is not necessary for an example.
Still, some additional description would be handy. I.e. momentum before absorption equals to momentum after, hence momentum after is zero. After absorption there is only electron left, so its momentum is zero. Hence its velocity is zero for non-zero mass.